scripts/files/file_tiff.py

```diff
@@ -36,15 +36,36 @@ def __init__(
         self._srs: Optional[bytes] = None
 
     def set_srs(self, srs: bytes) -> None:
+        """Set the srs.
+
+        Args:
+            srs: the srs in bytes
+        """
         self._srs = srs
 
     def set_scale(self, scale: int) -> None:
+        """Set the scale.
+
+        Args:
+            scale: the scale as `int`
+        """
         self._scale = scale
 
     def set_path_standardised(self, path: str) -> None:
+        """Set the standardised file path.
+
+        Args:
+            path: the path to the standardised file
+        """
         self._path_standardised = path
 
     def get_gdalinfo(self) -> Optional[GdalInfo]:
+        """Get the `gdalinfo` output for the file.
+        Run gdalinfo if not already ran.
+
+        Returns:
+            the `gdalinfo` output
```

Comment:
This returns `None` if there was a error?

Reply:
I'm going fix that in another PR